WalletWasabi/WabiSabi/Backend/Banning/CoinVerifierApiClient.cs

```diff
@@ -1,61 +1,61 @@
 using NBitcoin;
 using Newtonsoft.Json;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net;
 using System.Net.Http;
-using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
 using WalletWasabi.Logging;
 using WalletWasabi.WabiSabi.Backend.Statistics;
 
 namespace WalletWasabi.WabiSabi.Backend.Banning;
 
-public class CoinVerifierApiClient
+public class CoinVerifierApiClient : IAsyncDisposable
 {
-	public CoinVerifierApiClient(string token, Network network, HttpClient httpClient)
-	{
-		ApiToken = token;
-		Network = network;
-		HttpClient = httpClient;
-	}
-
-	public CoinVerifierApiClient() : this("", Network.Main, new() { BaseAddress = new("https://www.test.test") })
-	{
-	}
+	/// <summary>Maximum number of actual HTTP requests that might be served concurrently by the CoinVerifier webserver.</summary>
+	public const int MaxParallelRequestCount = 30;
 
-	private TimeSpan TotalApiRequestTimeout { get; } = TimeSpan.FromMinutes(3);
+	/// <summary>Maximum re-tries for a single API request.</summary>
+	private const int MaxRetries = 3;
 
-	private string ApiToken { get; set; }
-	private Network Network { get; set; }
+	/// <summary>Timeout for a single <see cref="SendRequestAsync(Script, CancellationToken)"/> invocation (including retries).</summary>
+	private static TimeSpan TotalApiRequestTimeout { get; } = TimeSpan.FromMinutes(3);
```

Comment:
Why `static`?

Reply:
Also, consider moving this property below the `ctor` or changing it to a `const` field, so we satisfy the CodeFactor.

Reply:
Oh I see, `TimeSpan` cannot be a `const`.